internal/generator/vector/output/security/security.go

```diff
@@ -95,6 +96,16 @@ func SecretPath(name string, file string) string {
 	return fmt.Sprintf("%q", filepath.Join("/var/run/ocp-collector/secrets", name, file))
 }
 
+func GetValueFromFile(name string, file string) string {
+	filename := SecretPath(name, file)
+	b, err := ioutil.ReadFile(filename)
+
+	if err != nil {
```

Comment:
We really need to stop writing functions that hide errors. I know we have a bunch of code that doesn't have good error reporting paths, but we need to fix that rather than perpetuate it. These errors should bubble up to where we can put them in a status message and log them so users can debug the problem.

Reply:
thanks @alanconway for suggestion. I agree that its not good practice. I was probably following other examples which also needs to be fixed.